kv/kvclient/rangefeed: TestRangefeedCatchupStarvation failed

kv/kvclient/rangefeed.TestRangefeedCatchupStarvation [failed](https://mesolite.cluster.engflow.com/invocations/default/813eb1c7-0280-4db8-ada8-5ed1c1ee01f0?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZjbGllbnQvcmFuZ2VmZWVkOnJhbmdlZmVlZF90ZXN0) on master @ [ccdd2647944cbbda82017879398e376cb9478363](https://github.com/cockroachdb/cockroach/commits/ccdd2647944cbbda82017879398e376cb9478363):


```
=== RUN   TestRangefeedCatchupStarvation
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestRangefeedCatchupStarvation1130508379
    test_log_scope.go:82: use -show-logs to present logs inline
    rangefeed_external_test.go:1988: -- test log scope end --
test logs left over in: outputs.zip/logTestRangefeedCatchupStarvation1130508379
--- FAIL: TestRangefeedCatchupStarvation (523.48s)
=== RUN   TestRangefeedCatchupStarvation/feed_type=mux/buffered_sender=false
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    rangefeed_external_test.go:1984: test timed out
--- FAIL: TestRangefeedCatchupStarvation/feed_type=mux/buffered_sender=false (389.42s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRangefeedCatchupStarvation.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56299

--- a/pkg/kv/kvclient/rangefeed/rangefeed_external_test.go
+++ b/pkg/kv/kvclient/rangefeed/rangefeed_external_test.go
@@ -1980,7 +1980,7 @@ func TestRangefeedCatchupStarvation(t *testing.T) {
 				if len(seen) >= totalKeys {
 					return
 				}
-			case <-time.After(testutils.DefaultSucceedsSoonDuration):
+			case <-time.After(testutils.SucceedsSoonDuration()):
 				t.Fatal("test timed out")
 			}
 		}

